### DISCUSSION TITLE
scalable treasury utilizing popular offchain defi revenue streams to fund treasury with liquidity

### DISCUSSION THREAD

**Discussion by storypup** [ethereumclassic/ECIPs#446] [Ideas]
This would leverage already existing projects in defi, methods that are typically used to generate revenue for groups or projects by putting a pool of liquidity to work, not generating extra costs for anyone in this community, and decentralizing control of the treasury funds into a dao-like managed treasury. In order to create a treasury like this without threatening the values of etc, I feel like it needs to start out not actually affecting ETC itself. Its just a revenue stream and a governance system (not tied to ETC protocol governance) that governs which projects applying for funds get funded, how much and which type of liquidity to keep on hand, funding bounties, and a very limited control over paying for its own development / security audits. Lets say a proposal gets passed but to fund the actual work, the devs wanting to work on the proposal also create a proposal to the treasury. This way they know beforehand if this is going to be a hard road, and roughly how much they are getting. Also allows for solo or freelance devs a platform for requesting funds as well. 

A future Possibility could be Anyone staking to yield pool generates governance tokens allowing them to vote on which projects/grants to fund. ) it also gives the chance for stackers and miners to win the pool if there treasury is well funded enough, or could create additional workstreams. 

This method is subjecting yourself to the risk of this type of pool.. Im not sure that there is much other than a lot of the features published by the org who's build im suggesting was only recently released for public use. I also dont have any concrete numbers to throw around yet. All i know is ive been in one of these pools and didnt lose my money, which is really the whole reason for this. No one reallu loses, but we are designing ways to generate funds and effectively use them. I dont think a tax is the answer although the prototreasury was a lot  better than the original proposal, and addressed some key issues. We should start with something very decentralized, very optional and see what grows from it. Its literally just a way to generate funds that doesnt require a marketing team this community doesnt have. We can keep things off chain, too.. but im banking on the fact that it actually does what is written here. 

The project i keep referencing that provides the core concept and build is called Pooltogether 3.0

https://docs.pooltogether.com/protocol/prize-splits

Judging by this, instead of awarding everyone at random , some or all rewarded wallets could be specified beforehand with a specific amount. That way, we always get money into the treasury for funding the devs if people use this. Eventually people could create an indentity (testnet) in whatever governance platform the treasury used for funding proposals, tie that to a metamask or something and then send etc to the treasury dao contract, the dao would set up a stake for the equivalent stablecoin in the pool and if you wanted to get fancy a UI could track that. 10% of profit could go to future dev and bounties which require treasury proposals. (im assuming yearn and the like work). 

I don’t fully understand the tech im talking about here.. my understanding has led me here. Hoping theres at least something useful here that isn’t a complete fantasy. Please pick this apart and find something useful if there’s anything good here.  

I realize this is complex, and maybe doesnt need to be at first..but it could be scaled down. Check it out.. the majority of my inspiration depends on utilizing other projects and development. I really think we are thinking too hard. Lets just focus on getting sustainable income that isnt fits a community project.   

There might also need to be an initial sponsor willing to take some risk here to experiment.. and devs who believe in it enough that they would be willing to get payed only after the pool is running.

**Comment by bobsummerwill** [comment]
I'm afraid that there isn't much usable here.

Total fees and activity on ETC are nowhere near enough for anything to be funded using this kind of indirect approach.

"There might also need to be an initial sponsor willing to take some risk here to experiment"

Right, and I such no obvious candidates for this.   That is the root issue for ETC.

"and devs who believe in it enough that they would be willing to get payed only after the pool is running."

And no obvious candidates who would be willing to take a leap of faith on this.

**Reply by storypup** [reply]
Appreciate your input Bob. In the couple of months since I wrote this I've come to realize what you say is the reality of ETC development and agree no one would take the time to set up something like this.

@bobsummerwill as a followup, I'd simply like to suggest implementing a simple and direct way for a community member to donate to a development fund. Even if it seems unlikely, there are those that recognize Etc coop does contribute to development and is historically transparent with their finances. I dont think its shameful or unreasonable for a nonprofit such as yours to simply advertise a short message giving a means of donating and to assure donors that contributions of any amount are welcome and appreciated. Id be more than happy to advocate this to the moderators of discord.
